test/regress/CMakeLists.txt

```diff
@@ -2744,6 +2744,21 @@ set(regression_disabled_tests
   regress3/issue4476-ext-rew.smt2
 )
 
+#-----------------------------------------------------------------------------#
+# Tests to exercise '--use-approx'
+set(use_appox_tests
+  regress0/use_approx/bug812_approx.smt2
+  regress0/use_approx/error0_approx.smt2
+  regress0/use_approx/issue2429_approx.smt2
+  regress0/use_approx/issue4714_approx.smt2
+  regress0/use_approx/siegel-nl-bases_approx.smt2
+  regress0/use_approx/specsharp-WindowsCard.15.RTE.Terminate_System.Int32_approx.smt2
+)
+
+if(USE_GLPK)
+  set(regress_0_tests ${regress_0_tests} ${use_appox_tests})
+endif()
```

Comment:
Instead of using CMake for this, please add a `; REQUIRES: glpk` at the top of the regression tests. That's the mechanism that we generally use to conditionally enable/disable tests depending on the features that cvc5 was compiled with.

Reply:
Ah! Certainly! I didn't know about that!

However, these tests don't *need* GLPK, right? `cvc5` might use "do more" if GLPK is enabled, but I think it still does something if you have `--use-approx` and not a GLPK build.

Basically, I agree with you, I'm just making sure you still agree with me, after you're aware of the above.

Reply:
It looks like is [always](https://github.com/cvc5/cvc5/blob/e4e19cd62b3eebed2de5b9b627509df0ffec23e1/src/theory/arith/theory_arith_private.cpp#L1884-L1885) [checked](https://github.com/cvc5/cvc5/blob/e4e19cd62b3eebed2de5b9b627509df0ffec23e1/src/theory/arith/theory_arith_private.cpp#L2969) in conjunction with [`ApproximateSimplex::enabled()`](https://github.com/cvc5/cvc5/blob/2283ee3b0327441c29caf26be977c1e4cd13c637/src/theory/arith/approx_simplex.cpp#L545-L551), so it won't have an effect if GLPK is disabled. Have you found a different location where this is not the case?

Reply:
Nope; not at all.

I guess that's my original point: `cvc5` does "something" if you use `--use-approx` **without** GLPK, but I am not entirely sure what that behaviour is ...

For example, if you *don't* have GLPK, is it possible we might get `unknown` where we get `sat`/`unsat` in the tests I added?